controllers/hetznerbaremetalhost_controller.go

```diff
@@ -170,6 +170,18 @@ func (r *HetznerBareMetalHostReconciler) reconcileSelectedStates(ctx context.Con
 	case infrav1.StateNone:
 		var needsUpdate bool
 		if !bmHost.DeletionTimestamp.IsZero() && bmHost.Spec.ConsumerRef == nil {
+			conditions.MarkTrue(
+				bmHost,
+				infrav1.DeletionInProgressCondition,
+			)
+			conditions.MarkFalse(
+				bmHost,
+				infrav1.CredentialsAvailableCondition,
+				infrav1.HetznerSecretUnreachableReason,
```

Comment:
is `HetznerSecretUnreachableReason` the correct reason?

Reply:
not sure , what do would you suggest it to be ?

Reply:
I would set it like below:

```
			conditions.MarkFalse(
				bmHost,
				infrav1.DeletionInProgressCondition,
				infrav1.DeletionInProgressReason,
				clusterv1.ConditionSeverityInfo,
				"Deletion in progress",
			)
```

Reply:
should I create a `CredentialsAvailableReason` for this then ?
cc @janiskemper